assets/js/components/dashboard-details/dashboard-details-app.js

```diff
@@ -45,6 +46,7 @@ class DashboardDetailsApp extends Component {
 		return (
 			<Fragment>
 				<Header />
+				{ featureFlags.widgets.pageDashboard.enabled && <WidgetContextRenderer slug="pageDashboard" /> }
```

Comment:
This should rather go above the `<DashboardDetailsModules />` further below, which is the component that it eventually will replace.

Reply:
Updated.